pyproject.toml

```diff
@@ -18,13 +18,17 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools]
 packages = ["torchqdynamics"]
 
-[tool.black]
-line-length = 80
-preview = true
-target-version = ["py38"]
-skip-string-normalization = true
+[tool.yapf]
+based_on_style = "facebook"
+column_limit = 88
+indent_width = 4
+SPLIT_BEFORE_NAMED_ASSIGNS = false
+DEDENT_CLOSING_BRACKETS = true
+SPLIT_ARGUMENTS_WHEN_COMMA_TERMINATED = false
+INDENT_DICTIONARY_VALUE = true
```

Comment:
I think these three guys are by default in the `"facebook"` style, so unnecessary to specify.

Reply:
The top and bottom ones yes, but not the middle one. Changing accordingly.

Reply:
The middle one is in PEP8 style from which Facebook inherits.